targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_EP_AGORA/PinNames.h

```diff
@@ -0,0 +1,315 @@
+/*
+ * PinNames.h
+ *
```

Comment:
Please add SPDX + license  text to new files - review platform files  for examples